### PR TITLE
refactor: move list of LLMQ types to its own file

### PR DESCRIPTION
**Pull request by thephez**

### CODE REVIEW

**Review by QuantumExplorer — APPROVED** [review]

File: dip-0006/llmq-types.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+These are the currently supported LLMQ and DKG types. Each row is a single type. Each column refers to a parameter found
+in [Parameters/Variables of a LLMQ and DKG](../dip-0006.md#parametersvariables-of-a-llmq-and-dkg).
+
+| name | quorumType | quorumSize | quorumMinSize | quorumThreshold | quorumDkgInterval | quorumDkgPhaseBlocks | quorumDkgBadVotesThreshold | quorumSigningActiveQuorumCount | Notes |
+|--|--|--|--|--|--|--|--|--|--|
+| LLMQ_50_60 | 1 | 50 | 40 | 30 (60%) | 24 (1 Hour) | 2 | 40 | 24 | |
+| LLMQ_400_60 | 2 | 400 | 300 | 240 (60%) | 288 (12 Hours) | 4 | 300 | 4 | |
+| LLMQ_400_85 | 3 | 400 | 350 | 340 (85%) | 576 ( 24 Hours) | 4 | 300 | 4 | |
```
Comment (UdjinM6) [inline dip-0006/llmq-types.md:8]: nit: I know it's meant to be a move-only PR but let's maybe also fix this tiny issue while we are at it
```suggestion
| LLMQ_400_85 | 3 | 400 | 350 | 340 (85%) | 576 (24 Hours) | 4 | 300 | 4 | |
```

**Review by UdjinM6 — APPROVED** [review]
👍 
(just one suggestion, can be ignored and fixed later)

File: dip-0006/llmq-types.md (hunk 1 above)
Comment (thephez) [reply dip-0006/llmq-types.md]: Eh, it's not code. Might as well fix it now :slightly_smiling_face:

**Review by UdjinM6 — APPROVED** [review]
👍

**Pull request merged**